cmd/root.go

```diff
@@ -303,15 +309,34 @@ func runRoot(cmd *cobra.Command, args []string) {
 
 	// publish initial settings
 	valueChan <- util.Param{Key: keys.EEBus, Val: conf.EEBus.Configured()}
-	valueChan <- util.Param{Key: keys.Hems, Val: conf.HEMS}
 	valueChan <- util.Param{Key: keys.Shm, Val: conf.SHM}
 	valueChan <- util.Param{Key: keys.Influx, Val: conf.Influx}
 	valueChan <- util.Param{Key: keys.Interval, Val: conf.Interval}
 	valueChan <- util.Param{Key: keys.Messaging, Val: conf.Messaging.Configured()}
 	valueChan <- util.Param{Key: keys.ModbusProxy, Val: conf.ModbusProxy}
 	valueChan <- util.Param{Key: keys.Mqtt, Val: conf.Mqtt}
 	valueChan <- util.Param{Key: keys.Network, Val: conf.Network}
-	valueChan <- util.Param{Key: keys.Sponsor, Val: sponsor.Status()}
+	valueChan <- util.Param{Key: keys.Sponsor, Val: struct {
+		sponsor.Status
+		FromYaml bool `json:"fromYaml"`
+	}{
+		Status:   sponsor.GetStatus(),
+		FromYaml: fromYaml.sponsor,
+	}}
+
+	var maxPower float64
+	if hems != nil {
+		maxPower = hems.MaxPower()
+	}
+	valueChan <- util.Param{Key: keys.Hems, Val: struct {
+		Config   globalconfig.Hems `json:"config"`
+		MaxPower float64           `json:"maxPower"`
+		FromYaml bool              `json:"fromYaml"`
+	}{
+		Config:   conf.HEMS,
+		MaxPower: maxPower,
```

Comment:
```suggestion
		Status: hems.GetStatus(),
```

Then we can keep the hems specific logic/vars (`maxPower`) out of setup.go.

We could the same structure (`config` & `status`) for other modules (like influx, messaging, ...) as well.